docs/index.html

```diff
@@ -29,11 +29,11 @@ <h1>CPAN Dashboard</h1>
 
     <h2>Current authors</h2>
     <ul>
-      <li><a href="/BDFOY/">BDFOY</a></li>
```

Comment:
I guess that changes like this are to make local testing earlier. Have you seen App::HTTPThis? I use it for all my local web testing. https://metacpan.org/pod/distribution/App-HTTPThis/bin/http_this

Reply:
Well, I have a local mojo thing that does that. But, my initial thought was that people would be able to run this under their own GitHub Pages and see the same result. The CNAME shenanigans kinda killed that, so I ended up setting up a domain for my fork and don't have to deal with extra stuff in the URL.